src/content/developers/docs/nodes-and-clients/node-architecture/index.md

```diff
@@ -0,0 +1,59 @@
+
+The execution client is responsible for transaction handling, transaction gossip, state management and supporting the Ethereum Virtual Machine ([EVM](/developers/docs/evm/)). However, it is **not** responsible for block building, block gossiping or handling consensus logic. These are in the remit of the consensus client.
+
+The execution client creates execution payloads - the list of transactions, updated state trie, and other execution-related data. Consensus clients include the execution payload in every block. The execution client is also responsible for re-executing transactions in new blocks to ensure they are valid. Executing transactions is done on the execution client's embedded computer, known as the [Ethereum Virtual Machine (EVM)](/developers/docs/evm).
+
+The execution client also offers a user interface to Ethereum through [RPC methods](/developers/docs/apis/json-rpc) that enable users to query the Ethereum blockchain, submit transactions and deploy smart contracts. It's common for RPC calls to be handled by a library like [Web3js](https://web3js.readthedocs.io/en/v1.8.0/), [Web3py](https://web3py.readthedocs.io/en/v5/), or by a user-interface such as a browser wallet.
+
+In summary, the execution client is:
+
+- a user gateway to Ethereum
+- home to the Ethereum Virtual Machine, Ethereum's state and transaction pool.
+
+## What does the consensus client do? {#consensus-client}
+
+The consensus client deals with all the logic that enables a node to stay in sync with the Ethereum network. This includes receiving blocks from peers and running a fork choice algorithm to ensure the node always follows the chain with the greatest accumulation of attestations (weighted by validator effective balances). Similar to the execution client, consensus clients have their own P2P network through which they share blocks and attestations. 
+
+The consensus client does not participate in attesting to or proposing blocks - this is done by a validator, an optional add-on to a consensus client. A consensus client without a validator only keeps up with the head of the chain, allowing the node to stay synced. This enables a user to transact with Ethereum using their execution client, confident that they are on the correct chain.
+
+## Validators {#validators}
+
+Node operators can add a validator to their consensus clients if 32 ETH is the deposit contract. The validator client comes bundled with the consensus client and can be added to a node at any time. The validator handles attestations and block proposals. They enable a node to accrue rewards or lose ETH via penalties or slashing. Running the validator software also makes a node eligible to be selected to propose a new block.
+
+[More on staking](/staking/).
+
+## Components of a node comparison {#node-comparison}
+
+| Execution Client                                   | Consensus Client                                                 | Validator                    |
```

Comment:
The table wasn't rendering correctly in the preview deploy. I think it was due to spacing, if this is fixed this is good to go.

<img width="754" alt="Screenshot 2022-12-09 at 13 23 06" src="https://user-images.githubusercontent.com/62268199/206712025-e3a4b374-2e7b-42c0-9315-cd62b2f28566.png">

Reply:
it's  just missing the final |. Fixing now